src/actions/actionTypes.js

```diff
@@ -29,5 +30,6 @@ export default({
     ERROR_USER: 'ERROR_USER',
 
     /* Warehouse */
+    WAREHOUSE_ITEM: 'WAREHOUSE_ITEM',
     WAREHOUSE_ITEMS: 'WAREHOUSE_ITEMS'
```

Comment:
`WAREHOUSE_ITEM` is necessary to get an item by a specific ID, `WAREHOUSE_ITEMS` for paged items w/o specificity.  I don't think we can collapse these, but we might be able to remove duplication by using a util for single items (see below).